<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="mainFrame3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1041</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#MyLoginForm {
background: gray;
}


#mainFrame3 {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
QLabel {
font-weight: bold;
font-size: 15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="QTabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#MyLoginForm {
background: rgb(211, 255, 15);

}


QTabWidget:pane {
border: 4px solid gray;
border-radius: 10px;
}




#QTabWidget {
border: 4px solid gray;
border-radius: 40px;
background: rgb(51, 10, 255);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 10px;
}


QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 10;
padding: 7px;
font-size: 10px;
padding-left: 10px;
padding-right: 10px;
min-width: 50px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
QLabel {
font-weight: bold;
font-size: 13px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero voyageur</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Agence</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Destination</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>230</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_agence">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_destination">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_email">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>132</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 120px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
       </property>
       <property name="text">
        <string>Ajouter un voyageurs</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>-10</y>
         <width>391</width>
         <height>371</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#MyLoginForm {
background: gray;
}


#frame_3 {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 120px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
QLabel {
font-weight: bold;
font-size: 15px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <zorder>frame_3</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>le_num</zorder>
      <zorder>le_nom</zorder>
      <zorder>le_prenom</zorder>
      <zorder>LineEdit_agence</zorder>
      <zorder>LineEdit_destination</zorder>
      <zorder>LineEdit_email</zorder>
      <zorder>pushButton_ajouter</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QLineEdit" name="LineEdit_destination_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>230</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_num_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Agence</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_prenom_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_nom_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Destination</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_agence_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero voyageur</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_email_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifier">
       <property name="geometry">
        <rect>
         <x>162</x>
         <y>290</y>
         <width>132</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 120px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
       </property>
       <property name="text">
        <string>Modifier un voyageurs</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>-20</y>
         <width>381</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#MyLoginForm {
background: gray;
}


#frame_2 {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 120px;
max-width: 50px;
min-height: 30px;
max-height: 13px;
}
QLabel {
font-weight: bold;
font-size: 13px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <zorder>frame_2</zorder>
      <zorder>LineEdit_destination_2</zorder>
      <zorder>label_16</zorder>
      <zorder>LineEdit_num_2</zorder>
      <zorder>label_17</zorder>
      <zorder>label_18</zorder>
      <zorder>LineEdit_prenom_2</zorder>
      <zorder>LineEdit_nom_2</zorder>
      <zorder>label_19</zorder>
      <zorder>label_20</zorder>
      <zorder>LineEdit_agence_2</zorder>
      <zorder>label_21</zorder>
      <zorder>LineEdit_email_2</zorder>
      <zorder>pushButton_modifier</zorder>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero voyageur</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEdit_num_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>-20</y>
         <width>381</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#MyLoginForm {
background: gray;
}


#frame_4 {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 120px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
QLabel {
font-weight: bold;
font-size: 13px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>280</y>
          <width>152</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 140px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
        </property>
        <property name="text">
         <string>Supprimer un voyageurs</string>
        </property>
       </widget>
      </widget>
      <zorder>frame_4</zorder>
      <zorder>label_22</zorder>
      <zorder>LineEdit_num_3</zorder>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>370</y>
       <width>132</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 120px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Num</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prenom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Destination</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>380</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier par</string>
     </property>
    </widget>
    <widget class="QFrame" name="mainFrame2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#MyLoginForm {
background: gray;
}


#mainFrame2 {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
QLabel {
font-weight: bold;
font-size: 13px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>50</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Donner le mail du voyageur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LineEdit_num_4">
      <property name="geometry">
       <rect>
        <x>88</x>
        <y>80</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer un mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>100</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Saisir le contenu du mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LineEdit_num_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>132</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDialogButtonBox {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 120px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>580</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#MyLoginForm {
background: gray;
}


#progressBar {
border: 1px solid gray;
border-radius: 12px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 1px solid gray;
border-radius: 8px;
}



QLabel {
font-weight: bold;
font-size: 20px;
}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>501</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#MyLoginForm {
background: gray;
}


#frame {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
QLabel {
font-weight: bold;
font-size: 15px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>571</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Pictures/pdfl.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>142</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: rgb(189, 255, 244);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 130px;
max-width: 50px;
min-height: 25px;
max-height: 13px;
}</string>
     </property>
     <property name="text">
      <string>liste des agences (pdf)</string>
     </property>
    </widget>
    <zorder>label_25</zorder>
    <zorder>QTabWidget</zorder>
    <zorder>comboBox</zorder>
    <zorder>label</zorder>
    <zorder>mainFrame2</zorder>
    <zorder>progressBar</zorder>
    <zorder>frame</zorder>
    <zorder>pushButton</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
